inst/extdata/variable_details.csv

```diff
@@ -510,11 +510,11 @@ CCC_17A,CCC_17A_cat5_NA::a,cat,"cchs2005_p, cchs2007_2008_p, cchs2009_2010_p, cc
-CCC_181,CCC_181_cat2_NA::b,cat,"cchs2009_s, cchs2010_s, cchs2012_s",[CCC_181],,cat,NA::b,2,else,else,N/A,else,else,Alzheimer's disease or other dementia,Do you have Alzheimer's Disease or any other dementia?,,,,active,,
+CCC_181,CCC_181_cat2_1,cat,"cchs2009_s, cchs2010_s, cchs2012_s, cchs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2015_2016_i, cchs2017_2018_i","cchs2001_i::CCCA_181, cchs2003_i::CCCC_181, cchs2005_i::CCCE_181, cchs2015_2016_i::CCC_145, cchs2017_2018_i::CCC_145, [CCC_181]",ICES,cat,1,2,Yes alzheimer's disease or other dementia,Yes alzheimer's disease or other dementia,N/A,1,Yes,Alzheimer's disease or other dementia,Do you have Alzheimer's Disease or any other dementia?,,,,active,,
+CCC_181,CCC_181_cat2_2,cat,"cchs2009_s, cchs2010_s, cchs2012_s, cchs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2015_2016_i, cchs2017_2018_i","cchs2001_i::CCCA_181, cchs2003_i::CCC_181, cchs2005_i::CCCE_181, cchs2015_2016_i::CCC_145, cchs2017_2018_i::CCC_145, [CCC_181]",ICES,cat,2,2,No alzheimer's disease or other dementia,No alzheimer's disease or other dementia,N/A,2,No,Alzheimer's disease or other dementia,Do you have Alzheimer's Disease or any other dementia?,,,,active,,
+CCC_181,CCC_181_cat2_NA::a,cat,"cchs2009_s, cchs2010_s, cchs2012_s, cchs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2015_2016_i, cchs2017_2018_i","cchs2001_i::CCCA_181, cchs2003_i::CCC_181, cchs2005_i::CCCE_181, cchs2015_2016_i::CCC_145, cchs2017_2018_i::CCC_145, [CCC_181]",ICES,cat,NA::a,2,not applicable,not applicable,N/A,6,not applicable,Alzheimer's disease or other dementia,Do you have Alzheimer's Disease or any other dementia?,,,,active,,
+CCC_181,CCC_181_cat2_NA::b,cat,"cchs2009_s, cchs2010_s, cchs2012_s, cchs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2015_2016_i, cchs2017_2018_i","cchs2001_i::CCCA_181, cchs2003_i::CCC_181, cchs2005_i::CCCE_181, cchs2015_2016_i::CCC_145, cchs2017_2018_i::CCC_145, [CCC_181]",ICES,cat,NA::b,2,missing,missing,N/A,"[7,9]",don't know (7); refusal (8); not stated (9),Alzheimer's disease or other dementia,Do you have Alzheimer's Disease or any other dementia?,,,,active,,
+CCC_181,CCC_181_cat2_NA::b,cat,"cchs2009_s, cchs2010_s, cchs2012_s, cchs2001_i, cchs2003_i, cchs2005_i, cchs2007_2008_i, cchs2009_2010_i, cchs2011_2012_i, cchs2013_2014_i, cchs2015_2016_i, cchs2017_2018_i","cchs2001_i::CCCA_181, cchs2003_i::CCC_181, cchs2005_i::CCCE_181, cchs2015_2016_i::CCC_145, cchs2017_2018_i::CCC_145, [CCC_181]",ICES,cat,NA::b,2,else,else,N/A,else,else,Alzheimer's disease or other dementia,Do you have Alzheimer's Disease or any other dementia?,,,,active,,

```

Comment:
@caitlink12 Can you do the same as in [this comment](https://github.com/Big-Life-Lab/cchsflow/pull/140/commits/db122101b4616be897bba7504d1a33352759da97#r2291391462)